cmd/manager/main.go

```diff
@@ -21,8 +21,11 @@ import (
 	"flag"
 	"fmt"
 	"github.com/openebs/node-disk-manager/pkg/setup"
+	"github.com/openebs/node-disk-manager/pkg/upgrade"
+	"github.com/openebs/node-disk-manager/pkg/upgrade/preupgrade"
```

Comment:
Is there a need for a preupgrade package? Can it be part of a step under the upgrade itself?

Reply:
Actually, my aim is to create a package for each release versions. like `1.0 -> 1.1`. Since only preupgrade is available for the current release, structured like. I shall move it to upgrade pkg itself.